shared/ui/components/form/Form.js

```diff
@@ -0,0 +1,54 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+import styled from 'styled-components'
+import { Button } from '@aragon/ui'
+
+const Form = ({
+  children,
+  onSubmit,
+  submitText,
+  disabled,
+  errors,
+}) => {
+  return (
+    <React.Fragment>
+      <VerticalSpace />
+      {children}
+      <Button
+        style={{ userSelect: 'none', marginTop: '24px' }}
+        mode="strong"
+        wide
+        onClick={onSubmit}
```

Comment:
Why not use an actual `<form>` element with an `onSubmit` handler? That also lets us pass `className` through so the `Form` can be wrapped with `styled` (or equivalently, so we can use the `css=` prop with it).

```js
const Form = ({
   children,
   onSubmit,
   submitText,
   disabled,
   errors,
 }) => {
   return (
    <form onSubmit={onSubmit}>
      ...
      <Button type="submit" ...>

Reply:
Then the submit function has to have e.preventDefault(). However, I can use a `div` instead of `React.Fragment` and then pass the className to it.

Reply:
I think having an `e.preventDefault()` is worth it. This allows normal form behavior, such as pressing "enter" to submit while editing a form field, instead of clicking the button.

Reply:
I wasn't thinking that I could have the `preventDefault` inside the form's `onSubmit` function like so:
```
    <form
      className={className}
      onSubmit={(e) => {
        e.preventDefault()
        onSubmit()
      }}
   >
```
This saves having to include it in the functions that are passed to this form